backend/cmm_helpers.ml

```diff
@@ -1403,6 +1487,31 @@ let get_field_computed imm_or_ptr mutability ~block ~index dbg =
 
 (* Getters for unboxed int fields *)
 
+let get_field_unboxed_int8 mutability ~block ~index dbg =
+  let memory_chunk = Byte_signed in
+  if Arch.big_endian
+  then
+    Misc.fatal_error
+      "Unboxed int8 fields only supported on little-endian architectures";
+  (* CR layouts v5.1: We'll need to vary log2_size_addr to efficiently pack
+   * int8s *)
```

Comment:
To support packing, it won't be just varying `log2_size_addr` that will need to be done, more generally, we'll need to refactor most of the code to go from having field indexes everywhere as arguments, to have byte offsets (and the associated changes to `array_indexing`), but that's another topic.